tests/dummy/app/templates/components/file-tree.hbs

```diff
@@ -0,0 +1,10 @@
+<div style="padding-left:1em">
```

Comment:
@aaxelb please don't get carried away with this implementation. A more full-featured file tree will happen: https://openscience.atlassian.net/browse/EOSF-16